manifests/pyvenv.pp

```diff
@@ -31,6 +32,7 @@
   Stdlib::Filemode            $mode        = '0755',
   Array[Stdlib::Absolutepath] $path        = ['/bin', '/usr/bin', '/usr/sbin', '/usr/local/bin',],
   Array                       $environment = [],
+  Variant[Boolean,String[1]]  $prompt      = false,
```

Comment:
I feel like setting it to `true` does not make sense, i would rather:

```suggestion
  Optional[String[1]]  $prompt             = undef,
```

Reply:
I don't disagree. I just used Variant because it was the most common variant in existing code. I'll change and use the opportunity to add a test case that ensures --prompt is not used on python 3.5.